ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml

```diff
@@ -233,7 +233,7 @@ tests:
     workflow: openshift-e2e-libvirt
 - as: ocp-image-ecosystem-remote-libvirt-s390x
   cluster: build01
-  cron: 5 1 7 */12 *
+  cron: 0 0 * * 0
```

Comment:
for future reference - do you want to add a comment on what this translates to ?

Reply:
Yes, I do.  I'll update with human-readable descriptions in comments.  Good call.

Reply:
Unfortunately the comments are removed when running `make update`.  I think we'll have to go without :(